src/plugins/rollbar.js

```diff
@@ -36,10 +36,20 @@ const rollbar = new Rollbar(config);
 export default {
   install(app) {
     app.config.errorHandler = (error, vm, info) => {
-      rollbar.error(error, { vueComponent: vm, info });
+
       if (app.config.devtools) {
         console.error(error);
       }
+
+      const vueComponent = {
+        name: vm.$options?.name ?? 'AnonymousComponent',
+        props: vm.$props ?? {},
+        data: vm.$data ?? {},
+        attributes: vm.$attributes ?? {},
+        route: vm.$route ?? {},
+      };
```

Comment:
The fix is simply to hand-pick the data we want off the `vm`. This seems a good starting point for understanding the context of an error, and it falls back where data is missing